css/_uswds-theme-custom-styles.scss

```diff
@@ -453,7 +442,6 @@ abbr[title="required"] {
 }
 
 .site-subheading {
-  @include typeset-h6;
```

Comment:
Removed entirely, since moving to bottom caused some font issues. All it gets us is uppercasing, which I addressed in the previous PR

Reply:
OK, to be fair, it DOES change the spacing a little between the eyebrow and the `h1`, but not enough to be an issue